### DISCUSSION TITLE
It's hard to debug express middleware without names

### DISCUSSION THREAD

**Discussion by pahaz** [keystonejs/keystone#5413] [Feature requests]
Hi! Inside keystone sources we use `app.use(.., (req, res, next) => { .. }` for creating express middleware.

But if you try to debut it by ` DEBUG=express:router NODE_ENV=production yarn dev` command.
You will see something like:

```
express:router use '/' <anonymous>
express:router use '/admin/api' <anonymous>
```

It's hard to understand what's going during request handling.

```
  express:router <anonymous>  : /admin/ +0ms
  express:router <anonymous>  : /admin/ +0ms
```

It's better to named functions (`app.use(.., function name (req, res, next) { .. }`). It will help to debug express routing.

**Comment by bladey** [comment]
Thanks for the suggestion @pahaz.
